monai/transforms/utils.py

```diff
@@ -1516,5 +1519,32 @@ def convert_to_contiguous(data, **kwargs):
     return data
 
 
+def backup_meta(data: Mapping[Hashable, Dict], key: Hashable) -> Mapping[Hashable, Dict]:
+    """Create a copy of the meta data (e.g., `image_meta_dict`-> `image_orig_meta_dict`)"""
+    orig_meta_key = PostFix.orig_meta(key)
```

Comment:
I feel the `orig` is confusing here, here the newly generated thing is called `original`.
Maybe we can add a `new_key` arg to store the copied content?

Thanks.

Reply:
The idea was to be able to back up the original meta data such that transforms can update the current meta data, but we store what the original meta data looked like.